lte/gateway/Makefile

```diff
@@ -179,66 +179,28 @@ test_oai: build_common ## Run all OAI-specific tests
 
 # Catch all for c service tests
 # This works with test_dpi and test_session_manager
-test_%: stop build_common
+test_%: build_common
```

Comment:
Unclear to me what the purpose of the stop clauses was in our test environment. Can I remove? These makefile recipes use `sudo` which is not present in the docker environment.

Reply:
This done to avoid issues running pipelineD UT while pipelineD service is running.
so this needs to be done manually after this change.